aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -188,7 +192,7 @@ def _accelerate_workflow_actions(
                     actions += esbuild_no_deps
             else:
                 # Invalid workflow, can't have no dependency dir and no installation
-                raise EsbuildExecutionError(message="Lambda Builders encountered and invalid workflow")
+                raise EsbuildExecutionError(message="Lambda Builders encountered an invalid workflow")
```

Comment:
Should this error be more specific? The comment has more context which could be useful to help customers understand why the build failed. As a customer "invalid workflow" doesn't really tell me much.